sdk/core/core-https/api-extractor.json

```
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "types/src/index.d.ts",
+  "docModel": {
+    "enabled": false
+  },
+  "apiReport": {
+    "enabled": true,
+    "reportFolder": "./review"
+  },
+  "dtsRollup": {
+    "enabled": true,
+    "untrimmedFilePath": "",
+    "publicTrimmedFilePath": "./types/coreHttps.d.ts"
```

nit: for future downlevel dts work
```suggestion
    "publicTrimmedFilePath": "./types/latest/coreHttps.d.ts"
```

---

not really needed, could be done in future.

---

no time like the present. I also fixed up the build task to run downlevel-dts